README.md

```
@@ -2,18 +2,13 @@
-|-----------|----------|
-| `nvim/` | Neovim config (lazy.nvim, blink.cmp, fzf-lua, oil.nvim, tokyonight) |
-| `powershell/` | PowerShell profile |
-| `wezterm/` | WezTerm config |
-| `starship/` | Starship prompt config |
-| `vim/` | Vim config |
-| `bash/` | Bash config (Linux) |
-| `cc/` | Claude Code settings |
-| `scoop/` | Scoop package lock (`scoopfile.json`) |
-| `winget/` | Winget package lock (`packages.json`) |
-| `scripts/` | Setup and maintenance scripts |
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)